docs/zimlet-templates.md

```diff
@@ -0,0 +1,29 @@
+## Zimlet Templates
+
+Using Zimlet templates you can take full control of scaffolding your next Zimlet.
+
+```sh
+$: zimlet create default my-project
+```
+
+The above command pulls the template from [billneff79/zimlet-default-template](https://github.com/billneff79/zimlet-default-template) - the default template.
```

Comment:
https://github.com/Zimbra/zm-x-zimlet-template-default

Reply:
is there a difference between https://github.com/billneff79/zimlet-default-template and https://github.com/Zimbra/zm-x-zimlet-template-default?